app/src/test/java/org/simple/clinic/home/overdue/OverdueScreenControllerTest.kt

```diff
@@ -108,64 +108,42 @@ class OverdueScreenControllerTest {
   }
```

Comment:
The diff for this commit doesn't view very nicely because I deleted two tests in the middle and added a new test at the end. It might be better for the reviewer to view this file as a whole instead of as a diff.